commands/pr.go

```diff
@@ -234,14 +238,25 @@ func listPulls(cmd *Command, args *Args) {
 	flagPullRequestLimit := args.Flag.Int("--limit")
 	flagPullRequestFormat := args.Flag.Value("--format")
 	if !args.Flag.HasReceived("--format") {
-		flagPullRequestFormat = "%pC%>(8)%i%Creset  %t%  l%n"
+		flagPullRequestFormat = "%pC%>(8)%i%Creset  %t %cC%cs %l%n"
 	}
 
 	pulls, err := gh.FetchPullRequests(project, filters, flagPullRequestLimit, func(pr *github.PullRequest) bool {
 		return !(onlyMerged && pr.MergedAt.IsZero())
 	})
 	utils.Check(err)
 
+	checkStatus, err := gh.FetchPullRequestsCheckStatus(project, filters, flagPullRequestLimit)
```

Comment:
I assume that this review and the [previous review](https://github.com/github/hub/pull/2361#discussion_r347086665) are talking on the same subject of whether we should make the status to be opted-in so that either the API call is executed and/or the format is changed when the user stated that they want to see the status in the `--format` input, right? Any further information on this? :grin:

Reply:
Any further info on this, @mislav ? :grin: 
Should we implement Status in the form of `--format` flag option to then be able to:
1. Modify the `flagPullRequestFormat` and,
2. Also make the API request call to fetchStatus become optional

Looking forward to hearing from you! :slightly_smiling_face: